Python/optimizer.c

```diff
@@ -266,8 +263,15 @@ counter_optimize(
     }
     executor->executor.execute = counter_execute;
     Py_INCREF(self);
+    int oparg = 0;
+    while (instr->op.code == EXTENDED_ARG) {
+        oparg = (oparg << 8) | instr->op.arg;
+        instr++;
+    }
+    assert(instr->op.code == JUMP_BACKWARD);
+    oparg = (oparg << 8) | instr->op.arg;
```

Comment:
Don't we usually write this loop like this?

```suggestion
    int oparg = instr->op.arg;
    while (instr->op.code == EXTENDED_ARG) {
        instr++;
        oparg = (oparg << 8) | instr->op.arg;
    }
    assert(instr->op.code == JUMP_BACKWARD);
```

That way the most complex expression (left-shift and bitwise-or) only occurs once.

Ditto below.